foqus_lib/framework/optimizer/optimization.py

```diff
@@ -77,6 +78,6 @@ def run(self):
         except Exception as e:
             logging.getLogger("foqus." + __name__).exception(
                 "Exception in optimization thread")
-            self.ex = sys.exc_info()
-            self.msgQueue.put(traceback.format_exception(
-                self.ex[0], self.ex[1], self.ex[2]))
+#            self.ex = sys.exc_info()
+#            self.msgQueue.put(traceback.format_exception(
+#                self.ex[0], self.ex[1], self.ex[2]))
```

Comment:
Would rather not have commented-out code checked in, unless there's a clear plan on it becoming uncommented-out and brought be in.

Reply:
Ok, then i'll go ahead and remove the commented code lines.